src/styles/blocks/card.scss

```
@@ -0,0 +1,86 @@
+.card {
+  width: $cell-size;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-bottom: 46px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px ;
+  }
+
+  &__title {
+    font-weight: 500;
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-font--color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__revievs {
+    justify-content: space-between;
+    display: flex;
+    margin: 0 0 24px;
+
+    &-title {
+      display: flex;
+      align-items: center;
+      color: $main-font--color;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 0 16px;
+
+    &-title {
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &-sum {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-font--color;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    background: #00acdc;
+    border-radius: 5px;
+    border: none;
+    color: #fff;
+    text-transform: uppercase;
+    font-family: "Roboto", sans-serif;
```

inherit font-family from a parent
```suggestion
    font-family: inherit;
```